lib/Control/Control.js

```diff
@@ -1663,7 +1663,7 @@ Control.concat = function (ctor, props, instance) {
 
 	if (props.style) {
 		if (instance) {
-			str = (proto.style ? (proto.style + ';') : '') + (props.style + ';');
+			str = (proto.style ? proto.style : '') + props.style;
```

Comment:
Is the issue with `overrideComponents` directly related to your change here? If not, then I think this should be good to go. You're right that by running `normalizeCssStyleString` at the end of this conditional, we don't open ourselves up to the possibility of not having missing semicolons.